### PR TITLE
6 adaptive

### PR DESCRIPTION
**Pull request by Kingstar22**
https://kingstar22.github.io/frontend-course-2021/homeworks/nikolay.krasniuk_Kingstar22/6-adaptive

### CODE REVIEW

File: homeworks/nikolay.krasniuk_Kingstar22/6-adaptive/index.html [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+                    <a class="news__learn" href="#">
+                        <p class="news__learn-text">Learn More</p>
+                        <img class="news__learn-arrow" src="img/news/arrow.svg" alt="arrow_learn">
+                    </a>
+                </article>
+            </main>
+        </section>
+        <section class="clients">
+            <h2 class="clients__title">Our Clients</h2>
+            <main class="clients__content">
```
Comment (DrSmile444) [inline homeworks/nikolay.krasniuk_Kingstar22/6-adaptive/index.html:182]: This part is not adaptive on ipad. Could you make it wrap on small screens?

**Review by DrSmile444 — CHANGES_REQUESTED** [review]
Everything else looks fantastic for me! Well done!

File: homeworks/nikolay.krasniuk_Kingstar22/6-adaptive/index.html (hunk 1 above)
Comment (Kingstar22) [reply homeworks/nikolay.krasniuk_Kingstar22/6-adaptive/index.html:182]: done

**Review by DrSmile444 — CHANGES_REQUESTED** [review]
Sorry, while I was reviewing the site again, I noticed that some elements look wrong on big screens.

I'd like to ask you to review the following sections:
1) Header. It should not be stretched to the whole page;
2) Delivery process;
3) Apply for Ambassador;
4) Company news.

If you need any help in debugging, feel free to write me down in TG. ✌️

kingstar22 github io_frontend-course-2021_homeworks_nikolay krasniuk_Kingstar22_6-adaptive_ (1)

**Review by DrSmile444 — APPROVED** [review]
Well done! Thank you ✌️

**Pull request merged**